src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +19,13 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .flatMap(num -> Arrays.stream(num.split(",")))
```

Comment:
All magic numbers, strings, and symbols in your code should be constants.
[this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)
don't use short names, fix them everywhere
```suggestion
                .flatMap(line -> Arrays.stream(line.split(SEPARATOR)))
```